### ISSUE TITLE
A11y: Headings of the footer boxes not semantically labeled as headings

### ISSUE THREAD

**Issue by blacksheep--** [shopware/shopware#7039]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The headlines of the footer boxes (CSS selector .footer-column-headline) are not marked up as H4 HTML elements.

For blind people who work with a screen reader, this means that the unmarked headlines do not appear in the overview lists generated by the screen reader.

Even in the mobile version with the implementation as an ARIA disclosure element, the content blocks must be marked with a heading.

*_Expected behaviour:_*

For the sake of consistency and uniform design of all these texts, they must be marked with semantic tags (`h4` HTML elements).

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. navigate to the footer

![Image](https://github.com/user-attachments/assets/e9775d6c-d8ea-493f-a5da-26c5deaef4aa)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40812**



[created from NEXT-40812, comment 541353]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40812, comment 541554]

**Comment by tobiasberge** [comment]
Hi @blacksheep-- ,

sorry for the late reply. Having headlines for this looks reasonable, can you maybe elaborate why you are suggesting H4 elements?

The only thing I see here is that the page content itself is dynamic and it is not deterministic what headlines will be there. So it can happen that the page only has h1 and h2 and no h3. And then the footer has h4 and we are skipping 3. Or the page has levels down to h5 and the footer has h4.

But looking at other pages with similar footers, they have the same "issue". Some choose h6 to always be the "last". Some have h2 but before are also h3, h4 etc.

When the headline is seen only for the region `<footer>` itself I guess it could be also h2 🤔
